brion/plugin/compartmentReportHDF5.cpp

```diff
@@ -487,28 +499,25 @@ void CompartmentReportHDF5::_readMetaData()
         const auto& dims = _data->getSpace().getDimensions();
         if (dims.size() != 2)
             BRION_THROW("Bad report: data is not 2-dimensional")
-
         _sourceMapping.frameSize = dims[1];
     }
     catch (std::exception& e)
     {
-        BRION_THROW(std::string("Error opening compartment report: ") + std::string(e.what()));
+        throw std::runtime_error(
+            std::string("Error opening compartment report: ") + e.what());
     }
 }
 
 void CompartmentReportHDF5::_parseBasicCellInfo()
 {
-    const HighFive::Group reportGroup = _file->getGroup("report");
-    const HighFive::Group allGroup = reportGroup.getGroup("All");
-
     // This not only parses the GIDs, but also computes the cell offsets
     // and compartment counts per cell, with the proper reordering if needed.
-    const auto& mapping = allGroup.getGroup("mapping");
+    const auto& mapping = _file->getGroup("mapping");
     std::vector<uint32_t> gids;
     std::vector<size_t> offsets;
-    const auto& gidsDataSet = mapping.getDataSet("node_ids");
```

Comment:
Why the naming change in all data sets?

Reply:
We have added back the support to the old hdf5 implementation for reports (compartment and spikes), since the tests were made using files of such format. The implementation for the new sonata hdf5 format can be found in brion/plugin/compartmentReportHDF5Sonata

Reply:
I see!